### PR TITLE
Podify [ SpaceCommander]

### PR DESCRIPTION
**Pull request by EricMuller22**
This came up in a discussion with @justinseanmartin the other day. Rather than pulling this in as a submodule in newer projects, might be nice to have a pod.

CC @dnkoutso @alanf @dfed

### CODE REVIEW

File: SpaceCommanderDummy.m [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#import <Foundation/Foundation.h>
```
Comment (dfed) [inline SpaceCommanderDummy.m]: has this file been run through the formatter? ;)

**Comment by dfed** [comment]
Are the file renames breaking?

**Comment by dfed** [comment]
And it's def nice to have a pod. Thanks for doing this!

File: SpaceCommanderDummy.m (hunk 1 above)
Comment (EricMuller22) [inline SpaceCommanderDummy.m]: It actually hasn't :disappointed:

**Comment by dnkoutso** [comment]
LGTM.

File: SpaceCommanderDummy.m (hunk 1 above)
Comment (dfed) [inline SpaceCommanderDummy.m]: Let's do that

hypnotoad

File: SpaceCommander.podspec [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  s.author       = { "iOS Team" => "ios@squareup.com" }
+  s.source       = { :git => "https://github.com/square/spacecommander.git", :tag => s.version }
+  
+  s.ios.deployment_target = "6.0"
+  s.osx.deployment_target = "10.6"
+
+  # Use this file to overcome CocoaPods limitation that at least one source file must be specified.
+  s.source_files = "SpaceCommanderDummy.m"
+
+  s.preserve_paths = "Testing Support/**/*.m", "**/*.{sh,py}", "bin", "README.md"
```
Comment (alanf) [inline SpaceCommander.podspec]: why not preserve the path "**/*" (that's everything right)?

**Comment by alanf** [comment]
Thanks for podifying! Yes, the file renames will break preexisting installs.

File: SpaceCommander.podspec (hunk 2 above)
Comment (dfed) [inline SpaceCommander.podspec]: I think you'd want "**/**" if you actually want everything. "**" is recursive, "*" is one layer.

File: SpaceCommanderDummy.m (hunk 1 above)
Comment (EricMuller22) [inline SpaceCommanderDummy.m]: Done

**Comment by dfed** [comment]
GORM!

madman

(assuming @alanf doesn't mind the breaking rename. I don't have enough context on that part)

File: SpaceCommander.podspec (hunk 2 above)
Comment (EricMuller22) [inline SpaceCommander.podspec]: What do you think @dnkoutso

**Comment by alanf** [comment]
I do mind the breaking rename!

**Comment by EricMuller22** [comment]
Haha fixing
u-mad

**Comment by EricMuller22** [comment]
@alanf @dfed ready for another look

File: SpaceCommanderDummy.m (hunk 1 above)
Comment (alanf) [inline SpaceCommanderDummy.m]: Looks good, a couple more things, this file needs a Square copyright header (see the testing files for example). Also, while you're add it, can you rename this file and `@implementation` to  `CocoaPodsDummy`

File: SpaceCommander.podspec [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  s.author       = { "iOS Team" => "ios@squareup.com" }
+  s.source       = { :git => "https://github.com/square/spacecommander.git", :tag => s.version }
+  
+  s.ios.deployment_target = "6.0"
+  s.osx.deployment_target = "10.6"
+
+  # Use this file to overcome CocoaPods limitation that at least one source file must be specified.
+  s.source_files = "SpaceCommanderDummy.m"
+
+  s.preserve_paths = "**/**"
```
Comment (dfed) [inline SpaceCommander.podspec]: has this been tested? I was shooting in the dark before.

File: SpaceCommander.podspec [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
+Pod::Spec.new do |s|
+  s.name         = "SpaceCommander"
+  s.version      = "1.0.0"
+  s.summary      = "[ SpaceCommander] provides tools which enable you to commit Objective-C code to a git repository using a unified style format."
+  s.homepage     = "https://github.com/square/spacecommander"
+  s.license      = { :type => 'Apache', :file => 'LICENSE' }
```
Comment (dfed) [inline SpaceCommander.podspec]: This line should read `s.license  = 'Apache License, Version 2.0'` I believe

File: SpaceCommander.podspec [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  s.author       = { "iOS Team" => "ios@squareup.com" }
```
Comment (dfed) [inline SpaceCommander.podspec]: I don't think external people can email this address. I'd make it `s.authors  = 'Square'`

Comment (alanf) [inline SpaceCommander.podspec]: maybe even `Square, Inc.`

File: SpaceCommanderDummy.m (hunk 1 above)
Comment (EricMuller22) [inline SpaceCommanderDummy.m]: Gah, that's what we named it at first haha

**Comment by EricMuller22** [comment]
@dfed it linted both ways but I think we want `**/*` - if there are any wrinkles, I'll iron them out later today when my project consumes this pod.

**Comment by alanf** [comment]
LGTM

**Comment by dfed** [comment]
Woot! @EricMuller22 do you know how to push the public pod version?

**Comment by EricMuller22** [comment]
@dfed let's hold off, we're testing on the internal repo first - there's going to be a follow-up PR to clean up at least one oversight.

**Comment by dfed** [comment]
😎

**Comment by orta** [comment]
:+1: nice idea

**Pull request merged**